themes/default/content/what-is/what-are-kubernetes-secrets.md

```diff
@@ -0,0 +1,275 @@
+    $ kubectl describe pod backend-mongodb-776ff496b7-hmlrq --namespace mongodb
+    # ...
+    # other output not shown
+   # Environment:
+   #   BITNAMI_DEBUG:                    false
+   #   MONGODB_EXTRA_USERNAMES:          root
+   #   MONGODB_EXTRA_DATABASES:          admin
+   #   MONGODB_EXTRA_PASSWORDS:          <set to the key 'mongodb-passwords' in secret 'demo-dbcreds'>  Optional: false
+   #   MONGODB_ROOT_USER:                root
+   #   MONGODB_ROOT_PASSWORD:            <set to the key 'mongodb-root-password' in secret 'demo-dbcreds'>  Optional: false
+    # ...
+    ```
+
+5. Create a pod to test connectivity. Note this pod will have the `demo-dbcreds` secret mounted.
+
+    ```bash
+    $ cat <<EOF > mongo-conn-test.yaml
+    apiVersion: v1
+    kind: Pod
+    metadata:
+     name: mongo-conn-test
+    spec:
+     containers:
+     - name: mongo-conn-test
+       image: mongo:latest
+       command: ["/bin/sh", "-c"]
+       args:
+         - |
+           echo "Waiting for MongoDB to be ready..."
+           until mongosh --host=backend-mongodb.mongodb.svc.cluster.local -u root -p \$(cat /mnt/secrets/mongodb-root-password) --authenticationDatabase admin --eval "db.stats()" > /dev/null 2>&1; do
+             sleep 5
+           done
+           echo "Connected to MongoDB!"
+       volumeMounts:
+       - name: secret-volume
+         mountPath: "/mnt/secrets"
+     volumes:
+     - name: secret-volume
+       secret:
+         secretName: demo-dbcreds
+    EOF
+    ```
+
+    Update your host if using a different resource name or namespace. This examples uses `backend-mongodb.mongodb.svc.cluster.local`.
+
+6. Create the pod.
+
+    ```bash
+    $ kubectl apply -f mongo-conn-test.yaml   --namespace mongodb
+    ```
+
+7. Verify connectivity via the logs.
+
+    ```bash
+    $ kubectl logs mongo-conn-test --namespace mongodb
+    # Waiting for MongoDB to be ready...
+    # Connected to MongoDB!
+    ```
+
+8. Delete the test pod.
+
+    ```bash
+    $ kubectl delete pod mongo-conn-test  --namespace mongodb
+    ```
+
+### Conclusion
+
+Kubernetes secrets are crucial to many applications and we must have a solution to the lifecycle of these. Kubernetes Secrets is the native way to manage secrets. By following best practices and using Kubernetes secrets, you can ensure the security of your applications and sensitive data.
+
+Now that you're equipped with the knowledge of Kubernetes Secrets, take your cloud infrastructure management to the next level with Pulumi. Explore these key resources to deepen your understanding and enhance your implementation strategies:
+
+- **Advanced Secrets Management**: Discover how to efficiently manage sensitive data and secrets in your cloud applications. Dive into Pulumi's [Secrets Management guide](/blog/managing-secrets-with-pulumi/) for in-depth information on encrypting specific values for added security and ensuring that these values never appear in plain text in your state file​.
```

Comment:
Given that we have only a single bullet here, I would suggest reformatting it as just another paragraph instead of a list (it doesn't really make sense to have a list with one item).